### DISCUSSION TITLE
Do you have Scaler to scale AKS cluster on basis of CPU or other metric

### DISCUSSION THREAD

**Discussion by sanprasingh** [kedacore/keda#1126] [General] [answered]
Hi Team,
Pls. help on this ,Do you have Scaler to scale AKS cluster on basis of CPU or other metric  ?

Thanks,
Sanjeev Singh

**Comment by tomkerkhove** [comment, marked as answer]
This is coming in v2.0 where you can do this by using standard metrics: https://keda.sh/docs/2.0/concepts/scaling-deployments/

**Reply by sanprasingh** [reply]
But we are using V1.0 for pod scaling on basis of azure service bus topic scaler.

**Reply by tomkerkhove** [reply]
You can add it next to it if you migrate to 2.0, anything below 2.0 will not support this.

**Reply by sanprasingh** [reply]
can we archive pod scaling on basis of azure service bus topic scaler in v2.0 ? if yes, then can we user v2.0 for production workload ?

**Reply by tomkerkhove** [reply]
Oh I've misread your original question - Your question was can you scale AKS cluster (so nodes) with KEDA? If so, no but AKS has this through [cluster autoscaler](https://docs.microsoft.com/en-us/azure/aks/cluster-autoscaler).

We with KEDA are looking if we should integrate with it (https://github.com/kedacore/keda/issues/637).

Yes, you can scale Azure Service Bus Topic and we support that since v1.0 - https://keda.sh/docs/1.5/scalers/azure-service-bus/

**Reply by sanprasingh** [reply]
Thanks for you response.
